### DISCUSSION TITLE
React Router 7 is too slow on Cloudflare Worker

### DISCUSSION THREAD

**Discussion by cwagner22** [remix-run/react-router#12496] [Q&A]
If I deploy a simple example app I get about 110ms TTFB.
If I add more routes, with added third party libraries increasing my server.js bundle size to about 1.5MB, TTFB takes about 600ms.
(even for the same route)
Why is there such a big difference?

**Comment by Monu18765** [comment]
The issue is due to Cloudflare workers' cold start nature. I recommend you test it on a dedicated server like Railways.app
